doc/packages.md

```diff
@@ -26,7 +26,7 @@
 | developer/parser/bison		| 3.7.6			| https://ftp.gnu.org/gnu/bison/
 | developer/pkg-config			| 0.29.2		| https://pkg-config.freedesktop.org/releases
 | developer/versioning/git		| 2.31.1		| https://www.kernel.org/pub/software/scm/git https://git-scm.com/
-| developer/versioning/mercurial	| 5.7.1			| https://www.mercurial-scm.org/release/?M=D https://www.mercurial-scm.org/wiki/WhatsNew
+| developer/versioning/mercurial	| 5.8			| https://www.mercurial-scm.org/release/?M=D https://www.mercurial-scm.org/wiki/WhatsNew
```

Comment:
does mercurial actually have to be in core ?

Reply:
No, probably not. It's been there from before we took over OmniOS and was originally required to download the openjdk sources (and probably the opensolaris ones before that). We could look at moving it to extra for the next release.